configs/mender_convert_config

```diff
@@ -124,9 +124,11 @@ MENDER_CLIENT_INSTALL="y"
 #
 # This is used to fetch the correct binaries
 #
-# Valid values are "latest" (default), "master" or a specific version
-# MEN-6948 mender-convert is not yet integrated with latest mender-client 4.0.0
-MENDER_CLIENT_VERSION="3.5.2"
+# Valid values are "auto" (default), "latest", "master" or a specific version
+# If set to "auto" then the Mender Client 4.0 or later will be installed unless
+# the distribution is Ubuntu jammy or older or Debian bullseye or older;
+# the latest Mender 3.x client will be installed in those cases.
+MENDER_CLIENT_VERSION="auto"
```

Comment:
We should set `MENDER_CLIENT_VERSION` in CI to `auto` too (and take care of [this line](https://github.com/mendersoftware/mender-convert/blob/cd3d3067688ae34d3666d7f5327bdae42aaefeb6/.gitlab-ci.yml#L230)) so that we run tests with the C++ client as we were doing up until now.

Reply:
I don't get why you want to change the `MENDER_CLIENT_VERSION` to `auto` in the pipeline. Let's discuss it when you have time for it.

Reply:
I was wrong - letting it be `latest` will in effect install and test the C++ client